test/cmd/builds.sh

```diff
@@ -19,6 +19,18 @@ cat "${OS_ROOT}/Dockerfile" | oc new-build -D - --name=test
 oc get bc/test
 oc new-build --dockerfile=$'FROM centos:7\nRUN yum install -y httpd'
 oc get bc/centos
+oc new-build -D $'FROM openshift/origin:v1.1' --to-docker
+[[ $(oc get bc/origin --template '{{with .spec.output.to}}{{.kind}} {{.name}}{{end}}') == "DockerImage library/origin:latest" ]]
+oc delete all --all
+# Build from Dockerfile with output to ImageStreamTag
+oc new-build -D $'FROM openshift/origin:v1.1\nENV ok=1' --to origin-test:v1.1
+[[ $(oc get bc/origin-test --template '{{with .spec.output.to}}{{.kind}} {{.name}}{{end}}') == "ImageStreamTag origin-test:v1.1" ]]
+# Build from Dockerfile with output to DockerImage
+oc new-build -D $'FROM openshift/origin:v1.1\nENV ok=1' --to-docker --to openshift/origin:v1.1-test
```

Comment:
This line is falling in Jenkins even though we delete all on line 24... maybe the delete just takes too long? Looking for ideas how to make this better.